test/NuGet.Core.FuncTests/NuGet.Packaging.FuncTest/SigningTests/SignatureTrustAndValidityVerificationProviderTests.cs

```diff
@@ -2269,6 +2274,47 @@ private static byte[] GetResource(string name)
                 $"NuGet.Packaging.FuncTest.compiler.resources.{name}",
                 typeof(SignatureTrustAndValidityVerificationProviderTests));
         }
+
+        private static IDisposable TrustPrimaryRootCertificate(PrimarySignature signature)
+        {
+            using (var certificateChain = SignatureUtility.GetCertificateChain(signature))
+            {
+                return TrustRootCertificate(certificateChain);
+            }
+        }
+
+        private static IDisposable TrustPrimaryTimestampRootCertificate(PrimarySignature signature)
+        {
+            var timestamp = signature.Timestamps.FirstOrDefault();
+
+            if (timestamp == null)
+            {
+                return NoOpDisposable.Instance;
```

Comment:
Is this necessary? You could return null as `using (null) { ... }` is perfectly valid